src/mlpack/methods/reinforcement_learning/environment/continuous_mountain_car_V2.hpp

```diff
@@ -0,0 +1,224 @@
+/**
+ * @file continuous_mountain_car_V2.hpp
+ * @author Abhinav Sagar
+ *
+ * This file is an implementation of Continous Mountain CarV2 task:
```

Comment:
Does the MountainCarContinuous-v0 env you reference here include the velocity goal?